kythe/go/indexer/emit.go

```diff
@@ -933,6 +935,22 @@ func (e *emitter) writeRef(origin ast.Node, target *spb.VName, kind string) *spb
 		} else {
 			e.writeEdge(target, rule.VName, rule.EdgeOut)
 		}
+		if !e.fmeta[file] {
+			e.fmeta[file] = true
```

Comment:
I think we still want to do this only when `rule.EdgeOut == edges.Generates`.

Reply:
Done